ironfish-rust-nodejs/tests/demo.test.slow.ts

```diff
@@ -27,6 +27,13 @@ describe('Demonstrate the Sapling API', () => {
     expect(typeof key.spending_key).toBe('string')
   })
 
+  it('Should return phrase words from key input', () => {
```

Comment:
I love that you made a test here for this, it's something I've wanted us to do more. 🙌